forest/jasmine/traj2stats.py

```diff
@@ -99,13 +99,17 @@ def transform_point_to_circle(lat: float, lon: float, radius: float
     return transform(aeqd_to_wgs84, buffer)
 
 
-def get_nearby_locations(traj: np.ndarray) -> Tuple[dict, dict, dict]:
+def get_nearby_locations(
+    traj: np.ndarray, osm_tags: List[str] = ["amenity", "leisure"]
```

Comment:
Would it make sense to make an `OSMTag` enum like `Frequency` and add it to `constants.py`?

Also, make `None` the default value to avoid potential issue with a mutable default argument?

Reply:
yes that makes sense